src/nwb_datajoint/common/common_lab.py

```diff
@@ -3,22 +3,26 @@
 
 schema = dj.schema('common_lab')
 
+
 @schema
 class LabMember(dj.Manual):
     definition = """
-    lab_member_name: varchar(80)
+    lab_member_name: varchar(200)
     ---
-    first_name: varchar(80)
-    last_name: varchar(80)
+    first_name: varchar(200)
+    last_name: varchar(200)
     """
 
+    # NOTE that names must be unique here. If there are two neuroscientists named Jack Black that have data in this
+    # database, this will create an incorrect linkage. NWB does not yet provide unique IDs for names.
```

Comment:
good point. I wonder if we should bring in the datajoint user name (currently in `LabMemberInfo`) to `LabMember` and use it as a unique user ID